features/layout-instability.yml

```
@@ -1,7 +1,7 @@
-draft_date: 2024-09-09
-name: Layout Instability
-description: TODO
+name: Layout instability
+description: The `LayoutShift` interface provides a way to measure and report on the layout stability of web pages based on movements of the elements on the page.
```

Consistency with other features like this:

```suggestion
description: The `layout-shift` performance entry and `LayoutShift` API measures the layout stability of web pages based on movements of the elements on the page.
```